lib/internal/Magento/Framework/DB/Select.php

```diff
@@ -484,6 +512,30 @@ public function exists($select, $joinCondition, $isExists = true)
     }
 
     /**
+     * add SQL_NO_CACHE to bypass querycache usage
+     *
+     * @param bool $flag
+     * @return $this
+     */
+    public function sqlNoCache($flag = true)
+    {
+        $this->_parts[self::NO_CACHE] = (bool)$flag;
+        return $this;
+    }
+
+    /**
+     * add SQL_CACHE to enable querycache usage
+     *
+     * @param bool $flag
+     * @return $this
+     */
+    public function sqlUseCache($flag = true)
```

Comment:
Naming is not correct, method name must start from verb. Consistently with `useStraightJoin` it could be named `useSqlCache` or `setUseSqlCache`. Also, `useSqlCache(false)` does not seem really clear.

Are those renderers supposed to be used at the same time? If not, there should be only one renderer and setter like `setSqlCache(SQL_CACHE / SQL_NO_CACHE / null)`.

At first glance (http://dba.stackexchange.com/questions/14463/oracle-benchmarking-sql-no-cache) it looks like this feature is MySQL-specific, thus it must not be placed in generic `Select` class.